### ISSUE TITLE
Settings link missing accessibility label

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#1539]
### Description of problem

- The "Settings" navigation link in the "About Contacts" screen is missing the accessibility label.
- Update link color

### Screenshots and/or log output

![IMG_0085](https://github.com/bcgov/bc-wallet-mobile/assets/390891/5d22ccef-0d2f-4c43-9293-fce33a121c03)

### Environment
- [ ] Occurs on Android
- [x] Occurs on iOS

Build #:

1252

Android Device Model: 

iOS Device Model: 

### Severity
- [ ] High
- [ ] Medium
- [ ] Low
